requirements.txt

```diff
@@ -8,9 +8,9 @@ opt_einsum
 pathos>=0.2.7
 pylatexenc>=2.10
 pyscf>=2.0.1
-qiskit>=0.39.0,<1.0.0
+qiskit>=1.0.0
 recommonmark
-qiskit_ibm_runtime==0.20.0
+qiskit-ibm-runtime==0.20.0
 qiskit-aer==0.13.3
```

Comment:
I believe we can loosen the requirements for these packages — checked and it seemed to work fine locally.

Reply:
Specifically, ibm runtime and aer

Reply:
Thank you for reviewing! I will commit the change once the test is fixed.